cluster/repository/cluster_blackbox_test.go

```diff
@@ -106,3 +185,81 @@ func (s *clusterTestSuite) TestExists() {
 	err = s.repo.CheckExists(context.Background(), cluster.ClusterID.String())
 	require.NoError(s.T(), err)
 }
+
+func (s *clusterTestSuite) TestQueryOK() {
+	cluster1 := test.CreateCluster(s.T(), s.DB)
+
+	clusters, err := s.repo.Query(func(db *gorm.DB) *gorm.DB {
+		return db.Where("cluster_id = ?", cluster1.ClusterID)
+	})
+
+	require.NoError(s.T(), err)
+	require.Len(s.T(), clusters, 1)
+	test.AssertEqualClusters(s.T(), cluster1, &clusters[0])
+}
+
+func verifyClusters(t *testing.T, clusters []repository.Cluster) {
+	verifyCluster(t, clusters, &repository.Cluster{
```

Comment:
Instead of hardcoding the expected values in the tests you can check if the expected clusters match what we have in configuration. So, when we change anything in the configuration the test will still work and won't need any modification.